pyproject.toml

```diff
@@ -1,20 +1,21 @@
 [tool.poetry]
-name = "todo-app"
-version = "0.1.0"
-description = "A simple ToDo app for your DevOps apprenticeship"
-authors = []
+name="todo-app"
+version="0.1.0"
+description="A simple ToDo app for your DevOps apprenticeship"
+authors=[]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-flask = "^1.1.2"
-python-dotenv = "^0.14.0"
-requests = "^2.25.1"
-PyYAML = "^5.4.1"
-selenium = "^3.141.0"
-pytest = "^6.2.4"
+python="^3.7"
+flask="^1.1.2"
+python-dotenv="^0.14.0"
+requests="^2.25.1"
+PyYAML="^5.4.1"
+selenium="^3.141.0"
+pytest="^6.2.4"
```

Comment:
To actually take advantage of the `--no-dev` flag in your `poetry install` command in the Dockerfile, you should move selenium and pytest into the `dev-dependencies` section lower down this file.

You can just cut/paste the lines but in future you can add them in the correct place in the first place with e.g. `poetry add pytest --dev`

Reply:
Done, thanks.